Library/Homebrew/test/rubocops/text_spec.rb

```diff
@@ -307,7 +303,7 @@ class Foo < Formula
       RUBY
     end
 
-    it "when `\#{share}/foo` is used instead of `\#{pkgshare}`" do
+    it %Q(reports an offense if "\#{share}/<formula name>" is present) do
```

Comment:
Does this not work? Single quotes won't do string interpolation

```suggestion
    it 'reports an offense if "#{share}/<formula name>" is present' do
```

Reply:
That doesn't work as it's seen as broken interpolation, i.e. we have to explicitly escape the `#`

Reply:
Are you sure? When I test it out in `brew irb` it seems to work fine...

Reply:
I made a quick test ruby script:

```ruby
puts 'reports an offense if "#{share}/<formula name>" is present'
```

And then ran `brew ruby test.rb`:

```console
$ brew ruby test.rb
reports an offense if "#{share}/<formula name>" is present
```

Reply:
Sorry I meant that it works, but `brew style` sees it as broken

Reply:
```
Library/Homebrew/test/rubocops/text_spec.rb:304:8: W: [Correctable] Interpolation in single quoted string detected. Use double quoted strings if you need interpolation.
    it 'reports an offense if "#{share}/<formula name>" is present' do
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

Reply:
Ah, I see. That's interesting... If it were up to me I'd say let's disable that cop and allow it. But I don't feel strongly about that, so I think `%Q` is fine. Although, if we use `%q` instead no need to escape the `#`:

```suggestion
    it %q(reports an offense if "#{share}/<formula name>" is present) do
```

Reply:
Same error:

```
Library/Homebrew/test/rubocops/text_spec.rb:304:8: W: [Correctable] Interpolation in single quoted string detected. Use double quoted strings if you need interpolation.
    it %q(reports an offense if "#{share}/<formula name>" is present) do
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Ah well, that's unfortunate. I agree, then, this is the best option.